### ISSUE TITLE
 /opt/rocm/bin/rocminfo  hsa api call failure at line 900, file: /data/jenkins_workspace/compute-rocm-rel-2.2/rocminfo/rocminfo.cc. Call returned 4104

### ISSUE THREAD

**Issue by wrightpt** [ROCmSoftwarePlatform/tensorflow-upstream#353]
**System information**
- Ubutnu 18.04 fresh install

- TensorFlow installed from (source or binary): Debian repository (https://rocm.github.io/ROCmInstall.html#ubuntu-support---installing-from-a-debian-repository)

then:
1."sudo apt install rocm-libs miopen-hip cxlactivitylogger"
2. "pip3 install --user tensorflow-rocm"

- TensorFlow version: latest release with repository. 
- Python version: 3.6
- Installed using virtualenv? pip? conda?: pip
- Bazel version (if compiling from source):
- GCC/Compiler version (if compiling from source):
- ROCm/MIOpen version:
- GPU model and memory: RX 570 4 gig

**Describe the problem**

**Provide the exact sequence of commands / steps that you executed before running into the problem**

Usually when i switch to Python 3.6 interpreter in pycharm i get no issues with using Tensorflow but now:

"_ModuleNotFoundError: No module named 'tensorflow_'"

**Any other info / logs**
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.

uname -r
4.15.0-46-generic

/opt/rocm/opencl/bin/x86_64/clinfo 
ERROR: clGetPlatformIDs(-1001)

 /opt/rocm/bin/rocminfo
hsa api call failure at line 900, file: /data/jenkins_workspace/compute-rocm-rel-2.2/rocminfo/rocminfo.cc. Call returned 4104

When i try to run code:

> /usr/bin/python3 /home/c/.PyCharmCE2018.3/config/scratches/scratch.py
Downloading data from https://storage.googleapis.com/tensorflow/tf-keras-datasets/mnist.npz
11493376/11490434 [==============================] - 1s 0us/step
WARNING:tensorflow:From /home/c/.local/lib/python3.6/site-packages/tensorflow/python/ops/resource_variable_ops.py:435: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
WARNING:tensorflow:From /home/c/.local/lib/python3.6/site-packages/tensorflow/python/keras/layers/core.py:143: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
2019-03-14 17:47:59.165000: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 FMA
terminate called after throwing an instance of 'ihipException'
  what():  std::exception

Process finished with exit code 134 (interrupted by signal 6: SIGABRT)<

thank you

**Comment by wrightpt** [comment]
everything works now. I had to restart. Many thanks for everything
